### PR TITLE
[LPT] GNA: Pwl transformation

### PR DESCRIPTION
**Pull request by vzinovie**
### Tickets:
 - 67524

### CODE REVIEW

File: src/tests/unit/gna/ngraph/transformations/gna_pwl_LP_transformation.cpp [hunk 1]
```diff
@@ -0,0 +1,238 @@
+// Copyright (C) 2022 Intel Corporation
```
Comment (eshoguli) [inline src/tests/unit/gna/ngraph/transformations/gna_pwl_LP_transformation.cpp:1]: `src/tests/unit/gna/ngraph/transformations/gna_pwl_LP_transformation.cpp` => `src/tests/unit/gna/ngraph/transformations/low_precision/pwl_transformation.cpp` ?

File: src/tests/unit/gna/ngraph/transformations/gna_pwl_LP_transformation.cpp [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+        pwl->set_friendly_name("output");
+
+        auto result = std::make_shared<ngraph::opset8::Result>(pwl);
+        func_ = std::make_shared<ngraph::Function>(
+            ngraph::ResultVector{result},
+            ngraph::ParameterVector{input},
+            "PWLTransformation");
+    }
+#include "ngraph/pass/visualize_tree.hpp"
+    ngraph::pass::VisualizeTree("/home/vzinovie/work/model_dumps/actual.dot").run_on_model(func_);
```
Comment (eshoguli) [inline src/tests/unit/gna/ngraph/transformations/gna_pwl_LP_transformation.cpp:139]: please, remove

File: src/tests/unit/gna/ngraph/transformations/gna_pwl_LP_transformation.cpp [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+        },
+        {
+            {},
+            {2.f, 4.f, 8.f},
+            {3.f, 0.f, -3.f},
+            {-3.5f, 0.75f, 1.5f},
+            true
+        },
+    },
+};
```
Comment (eshoguli) [inline src/tests/unit/gna/ngraph/transformations/gna_pwl_LP_transformation.cpp:224]: please, add test cases when PWL transformation has to be ignored: per-channel quantization, empty dequantization...

**Review by eshoguli — COMMENTED** [review]
wip

**Review by eshoguli — APPROVED** [review]

File: src/plugins/intel_gna/transformations/low_precision/pwl_transformation.cpp [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+    ngraph::graph_rewrite_callback callback = [this](pattern::Matcher& m) {
+        auto op = m.get_match_root();
+        if (transformation_callback(op)) {
+            return false;
+        }
+        return transform(*context, m);
+    };
+
+    auto m = std::make_shared<ngraph::pattern::Matcher>(matcher, "PWLTransformation");
```
[the remark this answers is not in the document]
Comment (vzinovie) [reply src/plugins/intel_gna/transformations/low_precision/pwl_transformation.cpp:36]: @nkogteva thanks! Done

**Comment by akladiev** [comment]
This PR will be closed in 2 weeks in case of no activity.

**Comment by akladiev** [comment]
This PR was closed because it has been stalled for 2 week with no activity.
